Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandardV3/README.md

```diff
@@ -0,0 +1,4728 @@
+Endpoint Standard is an industry-leading next-generation antivirus (NGAV) and behavioral endpoint detection and response (EDR) solution. Endpoint Standard is delivered through the Carbon Black Cloud, an endpoint protection platform that consolidates security in the cloud using a single agent, console and data set.
+This integration was integrated and tested with version 7 of Carbon Black Endpoint Standard v3.
+### Migration to Carbon Black Endpoint Standard v3
+
+#### Deprecated Commands in Carbon Black Endpoint Standard v2
+
+The following commands from the Carbon Black Endpoint Standard v2 integration have been deprecated and replaced with v3 commands:
+
+| Deprecated Command | Replaced with v3 Commands | Comments |
```

Comment:
Instead of Replaced with v3 Commands, recommend the column title be:
New v3 Command